platform/android/java/editor/build.gradle

```diff
@@ -0,0 +1,104 @@
+// Gradle build config for Godot Engine's Android port.
+//
+// Do not remove/modify comments ending with BEGIN/END, they are used to inject
+// addon-specific configuration.
+apply from: 'config.gradle'
```

Comment:
You want this file to mirror the [`platform/android/java/plugins/godotpayment/build.gradle` build file](https://github.com/godotengine/godot/blob/master/platform/android/java/plugins/godotpayment/build.gradle) but using the `com.android.application` plugin instead of the `com.android.library` plugin.

The reason why is that the `app` module needs to be a standalone project when it's used as a custom build template, while the `editor` module will always be used within the context of the `platform/android/java` multiproject.

Reply:
Oh, I thought you meant for it to be completely different app

Reply:
Yes, that I meant, the `build.gradle` file just needs some cleanup. I've added https://github.com/thebestnom/godot/pull/1 to show what I meant.